IBPSA/Fluid/Geothermal/Aquifer/Data/Template.mo

```diff
@@ -0,0 +1,31 @@
+within IBPSA.Fluid.Geothermal.Aquifer.Data;
+record Template
+  "Template for soil data records"
+  extends Modelica.Icons.Record;
+  parameter Modelica.Units.SI.ThermalConductivity kSoi
+    "Thermal conductivity of the soil material";
+  parameter Modelica.Units.SI.SpecificHeatCapacity cSoi
+    "Specific heat capacity of the soil material";
+  parameter Modelica.Units.SI.Density dSoi(displayUnit="kg/m3")
+    "Density of the soil material";
```

Comment:
This data record should also include the porosity `phi` as it is a property of the soil, and not of the well model.

Reply:
Done